tests/unit/helpers/class-wc-mock-wc-data-store.php

```diff
@@ -109,7 +109,7 @@ public function update( &$object ) {
 		if ( 'user' === $this->meta_type ) {
 			wp_update_user(
 				[
-					'ID'         => $customer_id,
+					'ID'         => $customer_id, // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
I think that `$customer_id` is a legit undefined variable and should be fixed by replacing it with `$content_id`.

It's never caught because that `update()` function is never called but looking at the code, it's just conditionally calling `wp_update_user()` or `wp_update_post()` depending on the `meta_type`.

```suggestion
					'ID'         => $content_id,
```

Reply:
Thank you Shendy! ❤️